frontend/src/main/java/de/iav/frontend/model/AppUser.java

```diff
@@ -0,0 +1,21 @@
+package de.iav.frontend.model;
+
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.DBRef;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Document(collection = "users")
+public record AppUser(
+        String id,
+        @Indexed(unique = true)
+        String username,
+        @Indexed(unique = true)
+        String email,
+        String password,
+        AppUserRole role,
+        @DBRef
+        List<Ticket> tickets
```

Comment:
Die Mongo-Annotationen brauchen wir im FE nicht

Reply:
Wenn diese drin sind, solltest du auch in den Abhängigkeiten schauen, ob nicht etwas reingerutscht ist, dass du gar nicht brauchst iM FE